apps/src/codebridge/FileBrowser/dragAndDropUtils.ts

```diff
@@ -64,11 +69,12 @@ export const fileBrowserKeyboardCoordinateGetter: (
   });
   const currentIndex = orderedRects.indexOf(over?.id as string);
   let nextIndex = currentIndex;
-  // Find the next index based on the key pressed.
-  if (event.code === KeyboardCode.Down) {
-    nextIndex = Math.min(currentIndex + 1, orderedRects.length - 1);
+  // Find the next index based on the key pressed. We cycle around if they reach
+  // the beginning or end of the list.
+  if (event.code === KeyboardCode.Down || event.code === KeyboardCode.Tab) {
```

Comment:
Currently, shift+TAB  still moves index forward as TAB does which may seem counter-intuitive. What do you think about also checking for shift+TAB? I think @hannahbergam about this in [this thread](https://codedotorg.slack.com/archives/C06JELCAPT9/p1745362299991019?thread_ts=1745275490.059459&cid=C06JELCAPT9).

```
if (event.code === KeyboardCode.Down || (event.code === KeyboardCode.Tab && !event.shiftKey)) {...
} else if (event.code === KeyboardCode.Up || (event.code === KeyboardCode.Tab && event.shiftKey)) {...}

Reply:
Thanks!